source/common/ssl/connection_impl.cc

```diff
@@ -15,6 +15,20 @@
 namespace Envoy {
 namespace Ssl {
 
+namespace {
+// TODO(mattklein123): Currently we don't populate local address for client connections. Nothing
+// looks at this currently, but we may want to populate this later for logging purposes.
+Network::Address::InstanceConstSharedPtr
+getNullLocalAddress(const Network::Address::Instance& address) {
```

Comment:
I asked @hennna to un-factor this, since it's not a generally useful utility function, it's very specific to just these two modules as a workaround. Thoughts?